episodes/05-memory.md

```diff
@@ -9,7 +9,8 @@ In our final topic in how people learn (and therefore, how we can be more
 effective instructors), we will be learning more about human memory:
 specifically, how to remove unnecessary "load" in order to facilitate learning.
 
-::::::::::::::::::::::::::::::::::::::: objectives
+::::::::::::::::::::::::::::::::::::::: 
+objectives
```

Comment:
Somehow the fenced div got severed here. This should fix it. 

```suggestion
::::::::::::::::::::::::::::::::::::::: objectives
```

Reply:
@KwiatkowskiD here is a guide on how you can update your PR with this suggestion: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request